pkg/pmem-csi-driver/pmem-csi-driver.go

```diff
@@ -104,9 +98,6 @@ type Config struct {
 	NodeID string
 	//Endpoint exported csi driver endpoint
 	Endpoint string
-	//TestEndpoint adds the controller service to the server listening on Endpoint.
-	//Only needed for testing.
-	TestEndpoint bool
 	//Mode mode fo the driver
 	Mode DriverMode
 	//RegistryEndpoint exported registry server endpoint
```

Comment:
`RegistryEndpoint` no more needed and could be removed.

Reply:
Removed (but not pushed yet).